insolar/insolar
# INS-2310: healthcheck HTTP handler

**- What I did**
Added health check HTTP GET handle.

**- How I did it**
Checker reads all discovery nodes from certificated and check that _all_ of them are present in Working List.

**- How to verify it**
Start network, check `http://<node ip>:<API port>/healthcheck`, it should return `200 OK`. Kill one node and check again, it should return `500 FAIL`.
